build/NuSpecs/WindowsAppSDK-Nuget-Native.C.props

```diff
@@ -28,9 +28,12 @@
         %(AdditionalDependencies);
       </AdditionalDependencies>
     </Link>
-    <PostBuildEvent>
-      <Command>xcopy.exe /y "$(MSBuildThisFileDirectory)..\..\runtimes\win10-$(_WindowsAppSDKFoundationPlatform)\native\Microsoft.WindowsAppRuntime.Bootstrap.dll" "$(OutDir)"</Command>
-    </PostBuildEvent>
   </ItemDefinitionGroup>
 
+  <Target Name="CopyMicrosoftWindowsAppRuntimeBootstrapdllToOutDir" Condition="'$(AppxPackage)' != 'true'" AfterTargets="Build">
```

Comment:
>'$(AppxPackage)' != 'true'

Is that the right test or should it be `'$(WindowsPackageType)'=='None'` ?

@Scottj1s @bpulliam @evelynwu-msft what's the latest practices for "My VS project is not making an MSIX package"?

Reply:
This condition came from the ItemGroup above.